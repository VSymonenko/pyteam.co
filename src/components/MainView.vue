<template lang="pug">
    div(class='flex-row')
      v-spacer(class="hidden-sm-and-down" :style={width: '50%'})
      div
        slot
      v-spacer(class="hidden-sm-and-down" :style={width: '50%'})
</template>

<script lang="ts">
import Vue from 'vue';
import Component from 'vue-class-component';

@Component({})
export default class MainView extends Vue {}
</script>

<!-- Add "scoped" attribute to limit CSS to this component only -->
<style lang="stylus" scoped>
</style>
