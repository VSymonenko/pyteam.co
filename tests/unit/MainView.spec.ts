import MainView from '@/components/MainView.vue';
import { shallowMount } from '@vue/test-utils';
import './common';

test('renders correctly', () => {
  const wrapper = shallowMount(MainView);

  expect(wrapper.isVueInstance()).toBeTruthy();
  expect(wrapper).toMatchSnapshot();
});
