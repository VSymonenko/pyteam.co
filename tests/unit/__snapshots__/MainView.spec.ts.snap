// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div class="flex-row">
  <vspacer-stub class="hidden-sm-and-down"></vspacer-stub>
  <div></div>
  <vspacer-stub class="hidden-sm-and-down"></vspacer-stub>
</div>
`;
